docs/API.md

```diff
@@ -979,26 +979,26 @@ Example response:
 
 An object with the following properties:
 
-* `"username"`: An *username*. The name of the account of an authorized user.
-* `"api_key"`: An *api key*. The api_key associated with the username.
-* `"domains"`: A list of *domain names*. The domains to be tested.
-* `"test_params"`: As described below.
+* `"username"`: An *username*, required. The name of the account of an authorized user.
+* `"api_key"`: An *api key*, required. The api_key associated with the username.
+* `"domains"`: A list of *domain names*, required. The domains to be tested.
+* `"test_params"`: As described below, optional. (default with default value as below)
```

Comment:
> default with default value as below

This wording looks a bit confusing. I suggest this instead:

> default: `{}`

Reply:
done